src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age !== undefined
+        && <p className="Person__age">{`I am ${age}`}</p>
+      }
+      {isMarried
+        ? <p className="Person__partner">{`${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`}</p>
+        : <p className="Person__partner">I am not married</p>
+      }
```

Comment:
```suggestion
     <p className="Person__partner">
       {isMarried 
         ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}` 
          : I am not married}
       </p>
```
consider to make ternary just for text inside Person__partner element